packages/notifications-controller/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.0]
+
+### Added
+
+- Initial release ([#4320](https://github.com/MetaMask/core/pull/4320))
+
+[1.0.0]: https://github.com/MetaMask/core/releases/tag/@metamask/notifications-controller@1.0.0
```

Comment:
Instead of specifying a version here, it is customary to leave this changelog blank, but if you want to go ahead and provide a changelog entry that is fine too — just put this under an "Unreleased" header (you don't need to provide a PR number in this case):

``` suggestion
## [Unreleased]

- Initial release

[Unreleased]: https://github.com/MetaMask/core/

Reply:
addressed